### PR TITLE
Angular 2 boilerplate

### PR DESCRIPTION
**Pull request by suj97**

### CODE REVIEW

**Comment by suj97** [comment]
The bug was that I had copied some code from the angular.io site but it had error as it was not complete. Sliced it out.

File: basic/src/app/app.module.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { FormsModule } from '@angular/forms';
+import { HttpModule } from '@angular/http';
+
+import { AppComponent } from './app.component';
+
+import { MaterialModule } from '@angular/material';
+// other imports 
+@NgModule({
+  imports: [MaterialModule],
+  ...
```
Comment (kkimdev) [inline basic/src/app/app.module.ts]: I think you just pasted the exact tutorial from https://material.angular.io/guide/getting-started .  I'm pretty sure that `...` there means just other stuff.  Also, I don't think we're supposed to have a separate `@NgModule` section here. Probably you need to merge below.

This is why it's good to separate automatically generated one and manually edited one, it will make review a lot easier if reviewer knows where to look carefully (manually edited place).  So for this pull request, let's remove this section that you manually added, and make it as a separate pull request.

File: basic/src/app/app.module.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+})
+export class PizzaPartyAppModule { }
```
Comment (kkimdev) [inline basic/src/app/app.module.ts]: `PizzaPartyAppModule`? :) we're not building a Pizza Party app...

File: basic/src/index.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (kkimdev) [inline basic/src/index.html]: Let's make this line a separate pull request as it's a manually added line.

File: basic/src/app/app.module.ts (hunk 2 above)
Comment (suj97) [inline basic/src/app/app.module.ts]: Got you.

File: basic/src/app/app.module.ts (hunk 1 above)
Comment (suj97) [inline basic/src/app/app.module.ts]: Resolved.

File: basic/src/app/app.module.ts (hunk 2 above)
Comment (suj97) [inline basic/src/app/app.module.ts]: Resolved

File: basic/src/index.html (hunk 3 above)
Comment (suj97) [inline basic/src/index.html]: Resolved

**Comment by suj97** [comment]
Please have a look now.

**Comment by suj97** [comment]
The current state of my branch is the same as https://material.angular.io/guide/getting-started without any modification. @kkimdev
